examples/InformationPanel.horizontal.tsx

```diff
@@ -0,0 +1,85 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import {
+  Button,
+  InformationPanel,
+  InformationPanelBody,
+  InformationPanelContent,
+  InformationPanelHeader,
+  InformationPanelWrapper,
+  Input,
+  Label,
+  Table,
+  Text,
+  InputGrid,
+} from '@itwin/itwinui-react';
+import * as React from 'react';
+
+export default () => {
+  const [openRowIndex, setOpenRowIndex] = React.useState<number>(1);
+
+  return (
+    <InformationPanelWrapper>
+      <Table
+        columns={[
+          { id: 'name', Header: 'Name', accessor: 'name' },
+          {
+            Header: 'Details',
+            Cell: ({ row }) => (
+              <Button onClick={() => setOpenRowIndex(row.index)}>
+                Details
+              </Button>
+            ),
+          },
+        ]}
+        data={[...Array(3).fill(null)].map((_, index) => ({
+          name: `Row${index}`,
+        }))}
+        emptyTableContent='No data.'
+        style={{ minWidth: '450px' }}
+      />
+
+      <InformationPanel
+        isOpen={openRowIndex != undefined && openRowIndex !== -1}
+        orientation='horizontal'
+      >
+        <InformationPanelHeader onClose={() => setOpenRowIndex(-1)}>
```

Comment:
Should we show the use of `actions` in one of the examples? Or maybe in the code fence below the paragraph where you mention `actions`?

Reply:
what kind of actions do you want to show? if you have a small example in mind, i can add it.

Reply:
Maybe an "edit mode toggle" action, something similar to the [InformationPanel -> Custom actions story](https://itwin.github.io/iTwinUI/react/?story=informationpanel--custom-actions&theme=dark)? Since I guess disabled inputs are not preferrable, I guess we could replace them with `Text`s?

Reply:
i asked for "simple" 😅 edit mode is a whole beast that i'd rather not tackle here

Reply:
Ohh, then here are some simple actions I can think of to just demo the action buttons without much functionality:
* bookmark
* open model in the viewer/new tab
* info/help question mark (e.g. to show more info about what the complex fields mean)
* create sharable link
* upload more files (assuming the information panel is showing all the attached files)
* download/export
* upload

Reply:
good suggestions! i've added "open in new window" to both the static example and the horizontal one